CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 IMPROVEMENTS:
+* Set failurePolicy to Fail for connectInject mutating webhook so that pods fail to schedule when the webhook is offline. This can be controlled via `connectInject.failurePolicy` in helm.[[GH-1024](https://github.com/hashicorp/consul-helm/pull/1024)]
```

Comment:
```suggestion
* Set failurePolicy to Fail for connectInject mutating webhook so that pods fail to schedule when the webhook is offline. This can be controlled via `connectInject.failurePolicy`. [[GH-1024](https://github.com/hashicorp/consul-helm/pull/1024)]
```

Reply:
@ishustava how does this look?

Reply:
Looks good, thanks!